<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>A Multi-Omic Single Cell Brain Atlas of Mouse Neocortex</title>

<script src="site_libs/header-attrs-2.14/header-attrs.js"></script>
<script src="site_libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<style>h1 {font-size: 34px;}
       h1.title {font-size: 38px;}
       h2 {font-size: 30px;}
       h3 {font-size: 24px;}
       h4 {font-size: 18px;}
       h5 {font-size: 16px;}
       h6 {font-size: 12px;}
       code {color: inherit; background-color: rgba(0, 0, 0, 0.04);}
       pre:not([class]) { background-color: white }</style>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/pagedtable-1.1/css/pagedtable.css" rel="stylesheet" />
<script src="site_libs/pagedtable-1.1/js/pagedtable.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">
  code{white-space: pre-wrap;}
  span.smallcaps{font-variant: small-caps;}
  span.underline{text-decoration: underline;}
  div.column{display: inline-block; vertical-align: top; width: 50%;}
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
  ul.task-list{list-style: none;}
    </style>

<style type="text/css">code{white-space: pre;}</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>









<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
details > summary > p:only-child {
  display: inline;
}
pre code {
  padding: 0;
}
</style>


<style type="text/css">
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #adb5bd;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script type="text/javascript">
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.tab('show');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');

  // Navbar adjustments
  var navHeight = $(".navbar").first().height() + 15;
  var style = document.createElement('style');
  var pt = "padding-top: " + navHeight + "px; ";
  var mt = "margin-top: -" + navHeight + "px; ";
  var css = "";
  // offset scroll position for anchor links (for fixed navbar)
  for (var i = 1; i <= 6; i++) {
    css += ".section h" + i + "{ " + pt + mt + "}\n";
  }
  style.innerHTML = "body {" + pt + "padding-bottom: 40px; }\n" + css;
  document.head.appendChild(style);
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-inverse  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-bs-toggle="collapse" data-target="#navbar" data-bs-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">scBrainAtlas</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        <li>
  <a href="index.html">
    <span class="fa fa-home"></span>
     
    Home
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-dna"></span>
     
    UCSC Track Hub
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="ucscbrowser_mouse.html">Mouse</a>
    </li>
    <li>
      <a href="ucscbrowser_human.html">Human</a>
    </li>
  </ul>
</li>
<li>
  <a href="dolphinsuite.html">
    <span class="fa fa-laptop"></span>
     
    DolphinSuite
  </a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-project-diagram"></span>
     
    Cellxgene Browser
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="cellxgenebrowser.html">Mouse</a>
    </li>
  </ul>
</li>
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-envelope-o"></span>
     
    Contact
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="https://www.umassmed.edu/gtc/about/faculty-publications/miguel-sena-esteves/">Esteves Lab</a>
    </li>
    <li>
      <a href="https://www.umassmed.edu/biocore/">UMass Chan Med. Bioinformatics Core</a>
    </li>
    <li>
      <a href="https://alleninstitute.org/">Allen Institute for Brain Science</a>
    </li>
  </ul>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    <span class="fa fa-github"></span>
     
    GitHub
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
    <li>
      <a href="https://github.com/UMMS-Biocore/dolphinnext">DolphinNext</a>
    </li>
    <li>
      <a href="https://github.com/UMMS-Biocore/dmeta">DolphinMeta</a>
    </li>
    <li>
      <a href="https://github.com/UMMS-Biocore/dportal">DolphinPortal</a>
    </li>
  </ul>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div id="header">



<h1 class="title toc-ignore">A Multi-Omic Single Cell Brain Atlas of
Mouse Neocortex</h1>

</div>


<style>
body {
text-align: justify}
.center {
  display: block;
  margin-left: auto;
  margin-right: auto;
}
.main-container {
  max-width: 1200px;
  margin-left: auto;
  margin-right: auto;
}
</style>
<p><br /></p>
<p>We provide access to processed and analyzed data of human and mouse
single cell (<strong>scRNA-Seq</strong> and <strong>scATAC-Seq</strong>)
from large studies conducted in <strong>Allen Institute for Brain
Science</strong> as well as samples from additional projects such as
<strong>FANTOM5</strong> for <strong>CAGE-Seq</strong>. This integrative
project has been conducted in collaboration with <a
href="https://www.umassmed.edu/gtc/about/faculty-publications/miguel-sena-esteves/">Sena-Esteves
Lab</a> and <a href="https://www.umassmed.edu/biocore/">Bioinformatics
Core</a> at UMass Chan Medical School.</p>
<p>In collaboration with <a
href="https://www.umassmed.edu/gtc/about/faculty-publications/miguel-sena-esteves/">Sena-Esteves
Lab</a> and <a href="https://www.umassmed.edu/biocore/">Bioinformatics
Core</a> at UMass Chan Medical School, this website hosts a browser for
interactively visualizing transcriptional profiles of mouse brain cell
types, and two publicly accessible custom track hubs (for human and
mouse) within UCSC genome browsers.</p>
<p><strong>UCSC Track Hub:</strong> The scRNA-Seq, scATAC-Seq and
CAGE-Seq reads are individually aligned using predefined pre-processing
pipelines. Aligned reads are aggregated for cell types, tissues and
regions of interests before transformed into BigWig files. To provide
access to these aggregate single cell tracks we are hosting a custom
UCSC Track Hub (<a href="./ucscbrowser_mouse.html">Mouse</a> and <a
href="./ucscbrowser_human.html">Human</a>).</p>
<p><strong>Cellxgene Browser:</strong> The scRNA data on mouse neocortex
is analyzed using scRNA downstream analysis package called <a
href="https://satijalab.org/seurat/">Seurat</a>. To provide access to
the analyzed single cell data, we are hosting an interactive browser
based on <a href="./cellxgenebrowser.html">Cellxgene</a>.</p>
<p>All raw sequencing data (scRNA-Seq, scATAC-Seq and CAGE-Seq) were
processed by pipelines developed within the interactive pipeline manager
<a href="./dolphinsuite.html">DolphinNext</a> (<a
href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-020-6714-x">Yukselen
et. al 2020</a>). Metadata as well as the processed datasets are
currently being hosted in <a
href="./dolphinsuite.html">DolphinSuite</a>, an end-to-end
bioinformatics analysis platform.</p>
<p><br /></p>
<p><img width="100%" height="100%" src="images/index_human.png" class="center"></p>
<p><br /></p>
<div id="scrna-seq-tasic-et.-al-2018" class="section level5">
<h5><strong>scRNA-Seq (Tasic et. al 2018):</strong></h5>
<p>This study includes gene expression profiles of 23573 mouse neocortex
single cells (<a
href="https://www.nature.com/articles/s41586-018-0654-5">Tasic et. al
2018</a>) that are collected from either anterior lateral motor cortex
(<strong>ALM</strong>) or primary visual cortex (<strong>VISp</strong>)
of 360 mice donors whose ages are ranging between 51 and 91 days. Cells
are isolated into individual wells using fluorescence-activated cell
sorting (FACS) or manual picking. The cDNA libraries are generated and
amplified using SMART-Seq.</p>
<p>The paired sequencing reads of each individual cell are processed
using the <strong>scRNA-Seq (Tasic et al 2018)</strong> pipeline that
aligns and quantifies reads as previously described by (<a
href="https://www.nature.com/articles/s41586-018-0654-5">Tasic et. al
2018</a>).</p>
</div>
<div id="scatac-seq-mich-et.-al-2021" class="section level5">
<h5><strong>scATAC-Seq (Mich et. al 2021):</strong></h5>
<p>This study includes chromatin accessibility profiles of 3660 human
temportal cortex single cells (<a
href="https://linkinghub.elsevier.com/retrieve/pii/S2211-1247(21)00067-X">(Mich
et. al 2021</a>) that are collected primarily from the temporal cortex
(mostly from middle temporal gyrus) of 14 human donors whose ages are
ranging between 19 and 68. Cells are isolated into individual wells
using fluorescence-activated cell sorting (FACS) or manual picking.</p>
<p>The paired sequencing reads of each individual cell are processed
using the <strong>scATAc-Seq (Mich et al 2021)</strong> pipeline that
aligns and quantifies reads as previously described by (<a
href="https://linkinghub.elsevier.com/retrieve/pii/S2211-1247(21)00067-X">Mich
et. al 2021</a>).</p>
</div>
<div id="scatac-seq-graybuck-et.-al-2021" class="section level5">
<h5><strong>scATAC-Seq (Graybuck et. al 2021):</strong></h5>
<p>This study includes chromatin accessibility profiles of 3602 mouse
cortex single cells (<a
href="https://www.sciencedirect.com/science/article/pii/S0896627321001598?via%3Dihub">Graybuck
et. al 2021</a>) that are primarily collected from primary visual cortex
(<strong>VISp</strong>) of 60 mice donors whose ages are ranging between
38 and 81 days. Cells are isolated into individual wells using
fluorescence-activated cell sorting (FACS) or manual picking.</p>
<p>The paired sequencing reads of each individual cell are processed
using the <strong>scATAC-Seq (Graybuck et al 2021)</strong> pipeline
that aligns and quantifies reads as previously described by (<a
href="https://www.sciencedirect.com/science/article/pii/S0896627321001598?via%3Dihub">Graybuck
et. al 2021</a>).</p>
</div>
<div id="cage-seq-fantom5" class="section level5">
<h5><strong>CAGE-Seq (FANTOM5):</strong></h5>
<p><a href="https://fantom.gsc.riken.jp/5/">FANTOM5</a> project
incorporates CAGE sequencing reads from 1016 mouse and 701 human
samples. We have primarily aligned CAGE-Seq reads of <strong>primary
cells</strong> (neuronal, astrocyte, microglia etc.) and <strong>whole
tissue</strong> (cerebellum, visual cortex, hippocampus etc.) reads of
brain using a custom <strong>CAGE-Seq</strong> pipeline.</p>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.odd').parent('tbody').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open');
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
